worker/apiservercertwatcher/manifold.go

```diff
@@ -16,8 +16,16 @@ import (
 
 var logger = loggo.GetLogger("juju.worker.apiservercertwatcher")
```

Comment:
"apiservercertwatcher" yuk!

what is a servercer ?

Reply:
Can you expand more? This manifold has been around for a while.

Reply:
ah ok, i read the run together words wrongly! no spaces :-)